### PR TITLE
Rename loggable call function (#18)

### PR DESCRIPTION
**Pull request by augustingombault**
What?
-
Safe renaming of Loggable call function.

Why?
-
Use imperative tense for more comprehension.

### CODE REVIEW

**Comment by augustingombault** [comment]
https://github.com/drivy/backend-configs/issues/18

File: getaround_utils/spec/getaround_utils/mixins/loggable_spec.rb [hunk 1]
```diff
@@ -85,7 +92,7 @@ def append_infos_to_loggable(payload)
       it 'inject the class name' do
         expect(dummy_logger).to receive(:info)
           .with('message="dummy" key="value" origin="ChildClass"')
-        subject.use_loggable(:info, 'dummy', key: :value)
+        subject.loggable_log(:info, 'dummy', key: :value)
```
Comment (godric) [inline getaround_utils/spec/getaround_utils/mixins/loggable_spec.rb]: This should actually still be `use_loggable` which is defined in this spec, and verifies that it works when called (used) inside a class method.

Comment (augustingombault) [reply getaround_utils/spec/getaround_utils/mixins/loggable_spec.rb]: 🤦‍♂ 
Of course !!! My bad

**Comment by augustingombault** [comment]
@godric sorry, my bad. I fixed my misstakes !

File: getaround_utils/lib/getaround_utils/mixins/loggable.rb [hunk 2]
```diff
@@ -25,6 +25,14 @@ def base_loggable_logger
   end
 
   def loggable(severity, message, payload = {})
+    base_loggable_logger.send(
+      :warn,
+      "Deprecated usagage of GetaroundUtils::Mixins::Loggable.loggable(args*). Please use GetaroundUtils::Mixins::Loggable.loggable_log(args*) instead"
```
Comment (watsonbox) [inline getaround_utils/lib/getaround_utils/mixins/loggable.rb]: "usage"

Comment (watsonbox) [reply getaround_utils/lib/getaround_utils/mixins/loggable.rb]: I'm nitpicking, but I prefer the more idiomatic way of referring to instance methods with `#` 🙂 :

```ruby
GetaroundUtils::Mixins::Loggable#loggable
```

Comment (watsonbox) [reply getaround_utils/lib/getaround_utils/mixins/loggable.rb]: And I'm really nitpicking now, but did you mean `*args`?

Comment (augustingombault) [reply getaround_utils/lib/getaround_utils/mixins/loggable.rb]: Thks @watsonbox for all your comments

Comment (augustingombault) [reply getaround_utils/lib/getaround_utils/mixins/loggable.rb]: https://github.com/drivy/backend-configs/pull/20

**Pull request merged**